docs/ip-allowlist.md

```diff
@@ -0,0 +1,63 @@
+# GitHub IP Allow List Configuration
+
+If your organization is using [GitHub's Organization IP Allow List](https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-allowed-ip-addresses-for-your-organization), it needs to be configured properly for this Github app to be able to communicate with your organization's GitHub API.
+
+There are 2 methods to remedy this based on your company's security policy:
+
+1. a simple method which will enable _all_ GitHub apps with IPs specified in them to have access to your GitHub org's
+   APIs (_recommended_)
+2. a manual way by adding each CIDR ranges possible for this app to communicate from.
+
+**We recommend using the first method** as you only have to set it once and never have to think about it again. If there are
+any IP changes on our end in the future, we can just change the list on our end and it will automatically propagate to
+your organization. But for this to happen, you must trust every GitHub app installed or else risk a potential security
+breach by an app adding an attacker's IP to your allow list.
+
+If you'd like complete control over your IP Allow List, then you can enter the CIDR ranges manually in your GitHub 
```

Comment:
nit: "If you'd like complete control"  ->  "If you'd like to have a complete control"?